toolbox/linux/how.rst

```diff
@@ -0,0 +1,107 @@
+OK I'm Ready... How Do I Try/Install It?
+========================================
+
+So you're ready to install a Linux? Great! If it's your first time, hopefully
+you're starting with Ubuntu. Also if it's your first time, you may want to grab
+an experienced Linux buddy (ask in Slack or mailing list) to help you. Don't do
+this if you know you'll desperately need your computer the next day!
+
+*If you're not installing Ubuntu, these steps may not apply perfectly to you.*
+The information on virtual machines, live CDs, and live USBs will probably be
+helpful though.
+
+Step 1: Download
```

Comment:
I haven't read that section yet, but I'm assuming that there will be a "distros" section with various links to download the isos. This should probably link to that section.